### PR TITLE
#157608541 Create Utility CSS Classes for User Interface

### PR DESCRIPTION
**Pull request by holuborhee**
#### What does this PR do?
Create Utility Classes that can be used throught the user interfcace design

#### Description of Task to be completed?
- Create file `styles.css` to serve as external stylesheet for the whole UI
- Create a 13-grid Column system for Layout
- Add specific font-family for body and h1
- Create Utitlity Classes for Flexbox properties
- Add color variables for to the root of the page
- Add classes for padding and margin

#### How should this be manually tested?
After cloning create an html file and link to the stylesheet, then make a call to any of the classes.

#### Any background context you want to provide?
CSS Variables aren't supported in all browsers, as well as flexbox also. You can verify your browser support [here](https://caniuse.com/)

#### What are the relevant pivotal tracker stories?
[#157608541](https://www.pivotaltracker.com/story/show/157608541)

### CODE REVIEW

File: UI/styles.css [hunk 1]
```diff
@@ -0,0 +1,534 @@
...
+    --primary: #2196F3;
+    --secondary: #48BBFC;
+    --black: #3D3D3D;
+    --hover: #0475CE;
+    --white: #C4C4C4;
+}
+
+
+body {
+	background: #FAFAFA;
```
Comment (benfluleck) [inline UI/styles.css:11]: indentation for this file is off please use 2 spaces not a tab

File: UI/styles.css [hunk 2]
```diff
@@ -0,0 +1,534 @@
...
+}
+
+nav {
+	background: #F4F4F4;
+}
+
+
+/* 48BBFC - blue
+   2196F3 - 
+   3D3D3D - black
```
Comment (benfluleck) [inline UI/styles.css:28]: Remove the comments from your page

File: UI/styles.css [hunk 3]
```diff
@@ -0,0 +1,534 @@
...
+.mt4 {
+	margin-top: 1.5rem !important;
+}
+
+.mt5 {
+	margin-top: 3rem !important;
+}
+
+.mb1 {
+	margin-bottom: 0.25rem !important;
```
Comment (benfluleck) [inline UI/styles.css:58]: why are we doing so many important tags please read this article https://css-tricks.com/when-using-important-is-the-right-choice/. Please remove them

**Comment by benfluleck** [comment]
Your commit convention is wrong please refer to the naming convention docs in the channel. Also for just one file, you have way too many commits please limit it to one or two commits per branch. 

Also, I see little need for this file due to the fact that you can use both CSS grid and CSS flex together. Please have a look at CSS grid and see if it is something you can use effectively.

Thanks.

File: UI/styles.css (hunk 3 above)
Comment (holuborhee) [reply UI/styles.css:58]: Thanks for this link but I knew that `!important` is good for utility classes, and this is what the .*mb1 and .mb2* are, they are utility classes to apply a margin of a particular magnitude to any element. Also the post confirmed this.

image

**Pull request merged**